modules/deploy/Rakefile

```diff
@@ -37,12 +37,29 @@ task :setup_versions do
   @versions = YAML.load(version_yml)
 end
 
+task :setup_secrets do
+  @secrets = Hash.new
+  if not ENV["COUCHDB_USER"].nil?
```

Comment:
Rubyists seem to prefer `unless` over `if not`.

But I would invert the logic:

```
if thing.nil?
  puts "WARNING ..."
  thing = default value
else
  thing = ENV["thing"]
end
```